README.md

```diff
@@ -13,7 +13,7 @@ This project benchmarks the following minifiers:
 - [babel-minify](https://github.com/babel/minify/tree/master/packages/babel-minify) <sub>v0.5.2</sub>
 - [bun](https://github.com/oven-sh/bun) <sub>v1.1.29</sub>
 - [esbuild](https://github.com/evanw/esbuild) <sub>v0.24.0</sub>
-- [google-closure-compiler](https://github.com/google/closure-compiler-npm/tree/master/packages/google-closure-compiler) <sub>v20240317.0.0</sub>
+- [google-closure-compiler](https://github.com/google/closure-compiler-npm/tree/master/packages/google-closure-compiler) <sub>v20240317.0.0, advanced optimizations disabled</sub>
```

Comment:
This part is generated (between `<!-- minifiers:start -->` and `<!-- minifiers:end -->`) so this will get overwritten.